api/swagger-spec/v1.json

```diff
@@ -8882,14 +9026,6 @@
         "allowMultiple": false
        },
        {
-        "type": "string",
```

Comment:
We stopped accepting pointers to structs/interfaces as query parameters. I think that's why this is gone. In pkg/api/v1/types.go, sinceTime is a pointer to a struct, so, it no longer gets registered as a query parameter. I'm not sure if this matters or not, but, I think that's why it happened.